source/_components/rainbird.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Rainbird"
+description: "Instructions on how to integrate your Rainbird WiFi LNK within Home Assistant."

```

Comment:
**Spelling & Grammar**:
- `Rainbird WiFi LNK` -> `Rain Bird LNK WiFi` (according to the manufacturer)